### DISCUSSION TITLE
Infinite game vs finite game

### DISCUSSION THREAD

**Discussion by SimonBiggs** [LAION-AI/Open-Assistant#3398] [General]
Reading the following philosophy book from the 1980's makes me wonder if within its pages might be a key component of the solution to the AI alignment problem:

https://en.wikipedia.org/wiki/Finite_and_Infinite_Games#Summary

We are all so focused on finite games, we can't seem to look at the bigger picture of what the infinite game is. We need to find the infinite game ourself, and then help the AI be an infinite player not aiming to win, but instead aiming to have the continued play.

**Comment by PaulConyngham** [comment]
second this
